cmd/hil-vpn-privop/commands.go

```diff
@@ -0,0 +1,54 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+)
+
+// The actually functionality of each of the commands; the function
```

Comment:
Just a few typos in the comments in this block

Reply:
Fixed